onnx/external_data_helper.py

```diff
@@ -257,9 +262,18 @@ def write_external_data_tensors(model, filepath):  # type: (ModelProto, Text) ->
     @return
     The modified model object.
     """
+    any_not_save_external_tensor = False
     for tensor in _get_all_tensors(model):
         if uses_external_data(tensor):
-            save_external_data(tensor, filepath)
-            tensor.ClearField(str('raw_data'))
+            if tensor.HasField("raw_data"):
+                save_external_data(tensor, filepath)
+                tensor.ClearField(str('raw_data'))
+            else:
+                # If the model did not load external data, it's possible there is no raw_data
+                any_not_save_external_tensor = True
+
+    if any_not_save_external_tensor:
```

Comment:
not sure if this is needed

Reply:
After offline discussion, remove this warning